llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -184,22 +201,17 @@ splitModule(Module &M,
 
     // Save results.
     ResModules.push_back(std::move(MClone));
-    ResSymbolsLists.push_back(std::move(SymbolsList));
   }
 }
 
-// Saves specified collections of llvm IR modules and corresponding lists of
-// kernel names to files. Saves IR files list and TXT files list if user
-// specified corresponding filenames.
-static void saveResults(std::vector<std::unique_ptr<Module>> &ResModules,
-                        std::vector<std::string> &ResSymbolsLists) {
-  int NumOfFile = 0;
+// Saves specified collection of llvm IR modules to files.
+// Saves file list if user specified corresponding filename.
+static void
+saveResultModules(std::vector<std::unique_ptr<Module>> &ResModules) {
   std::string IRFilesList;
-  std::string TxtFilesList;
   for (size_t I = 0; I < ResModules.size(); ++I) {
     std::error_code EC;
-    std::string CurOutFileName = BaseOutputFilename + "_" +
-                                 std::to_string(NumOfFile) +
+    std::string CurOutFileName = BaseOutputFilename + "_" + std::to_string(I) +
```

Comment:
Is there an issue if the user is compiling `toto.cpp` and `toto_0.cpp` at the same time?
I guess it would be in 2 different temporary directories and thus no possible conflict?